hack-sound-server/meson.build

```diff
@@ -0,0 +1,48 @@
+namespace = 'HackSoundServer'
+library_name = 'hacksoundserver'
+export_package_name = 'hack-sound-server'
+
+api_version = '0'
+api_name = '@0@-@0@'.format(library_name, api_version)
+libtool_version = '0.0.0'
+
+hacksoundserver_dbus_sources = gnome.gdbus_codegen(
+    export_package_name,
+    sources: 'data/com.endlessm.HackSoundServer.xml',
+    interface_prefix : 'com.endlessm.',
+    autocleanup: 'all',
+    install_header: true,
+    install_dir: join_paths(get_option('includedir'), meson.project_name())
+)
+
+hacksoundserverlib = shared_library(
+    api_name,
+    hacksoundserver_dbus_sources,
+    version: meson.project_version(),
+    dependencies: [gio_dep, gio_unix],
+    install: true,
+    soversion: api_version,
+    version: libtool_version
```

Comment:
There are two `version` tags, as far as I undertand it's neither correct to use the project version or the libtool version here. Libtool is not used in Meson. Probably `soversion: '0'` and `version` left unspecified is okay, since if there are any backwards incompatible changes we will bump the `api_name` anyway.